setup.py

```diff
@@ -58,6 +58,7 @@
     "tabulate >= 0.9",
     "ipywidgets >=7.7.1",
     "humanize >= 4.6.0",
+    "matplotlib >= 3.7.1",
```

Comment:
Please add this to https://github.com/googleapis/python-bigquery-dataframes/blob/main/testing/constraints-3.10.txt too so that we know we always test against our advertised minimum version in at least one test session.

Reply:
`matplotlib==3.7.1` has been added into this file earlier. The `system-3.10` tests are not triggerred according to https://github.com/googleapis/python-bigquery-dataframes/blob/60594f4011ff72617932f37c1e53d4d3ba683ce4/noxfile.py#L57

If so, should I add the `matplotlib` to `contraints-3.9.txt` also?

Reply:
Oh, you're right. Yes add to `contraints-3.9.txt`. I was looking at the list of files sorted alphabetically.